NeurodataWithoutBorders/matnwb
# Check Component Order

## Motivation

Fixes #526 

## How to test the behavior?
1) Create a TimeSeries
2) Reference the TimeSeries in one row of TimeIntervals `timeseries`
3) export the file and run nwbinspector

There should not be a weird indexing-related hdmf error.
Ideally, MATLAB show throw the error before the file is written.

## Checklist
